LICENSE.md

```diff
@@ -0,0 +1,232 @@
+The contents of this repository are Copyright (c) corresponding authors and
+contributors, licensed under the `Permissive License Stack` meaning either of:
+
+- Apache-2.0 Software License: https://www.apache.org/licenses/LICENSE-2.0
+  ([...4tr2kfsq](https://gateway.ipfs.io/ipfs/bafkreiankqxazcae4onkp436wag2lj3ccso4nawxqkkfckd6cg4tr2kfsq))
+
+- MIT Software License: https://opensource.org/licenses/MIT
+  ([...vljevcba](https://gateway.ipfs.io/ipfs/bafkreiepofszg4gfe2gzuhojmksgemsub2h4uy2gewdnr35kswvljevcba))
```

Comment:
Whooops you copied the wrong one ;) 
We want the `dweb.link` ones, like here: https://raw.githubusercontent.com/ipfs/ipfs-repository-template/main/LICENSE.md

Reply:
ah, I got confused by the suggestion and thought it was the opposite 😆